<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComplexQTangoDemo</class>
 <widget class="QWidget" name="ComplexQTangoDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ComplexQTangoDemo</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="4">
    <widget class="TPushButton" name="tPushButton">
     <property name="text">
      <string>Switch States</string>
     </property>
     <property name="targets">
      <string>$1-&gt;SwitchStates;;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="TSimpleLabel" name="tSimpleLabel">
     <property name="text">
      <string>double_scalar</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="TLabel" name="tLabel_3">
     <property name="source">
      <string>$1-&gt;Status</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="TLabel" name="tLabel_2">
     <property name="source">
      <string>$1-&gt;State</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="TSimpleLabel" name="tSimpleLabel_2">
     <property name="text">
      <string>State:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QPushButton" name="pbMore">
     <property name="text">
      <string>More...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="6">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Inheriting QTangoComProxyReader / QTangoComProxyWriter: double_scalar</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="TDialRead" name="readDial">
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="notchesVisible">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="TDialWrite" name="writeDial">
        <property name="toolTip">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;This dial will change the &lt;span style=&quot; font-weight:600;&quot;&gt;short_scalar&lt;/span&gt; on the left&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="notchesVisible">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="leDialRead"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leDialWrite"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Using QTWatcher / QTWriter: double_scalar</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QSlider" name="horizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4">
    <widget class="TApplyNumeric" name="tApplyNumeric">
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>46</height>
      </size>
     </property>
     <property name="targets">
      <string>$1/double_scalar;;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="TLabel" name="tLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="source">
      <string>$1/double_scalar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TSimpleLabel</class>
   <extends>QLabel</extends>
   <header>TSimpleLabel</header>
  </customwidget>
  <customwidget>
   <class>TLabel</class>
   <extends>ELabel</extends>
   <header>TLabel</header>
  </customwidget>
  <customwidget>
   <class>TApplyNumeric</class>
   <extends>EApplyNumeric</extends>
   <header>TApplyNumeric</header>
  </customwidget>
  <customwidget>
   <class>TPushButton</class>
   <extends>QPushButton</extends>
   <header>TPushButton</header>
  </customwidget>
  <customwidget>
   <class>ELabel</class>
   <extends>QLabel</extends>
   <header>elabel.h</header>
  </customwidget>
  <customwidget>
   <class>EApplyNumeric</class>
   <extends>QWidget</extends>
   <header>eapplynumeric.h</header>
  </customwidget>
  <customwidget>
   <class>TDialRead</class>
   <extends>QDial</extends>
   <header>tdialread.h</header>
  </customwidget>
  <customwidget>
   <class>TDialWrite</class>
   <extends>QDial</extends>
   <header>tdialwrite.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
